zuul/tenants.yaml

```diff
@@ -21,6 +21,16 @@
           - ansible/ansible-runner
           - ansible/molecule
           - ansible-community/ansible-bender
+          - ansible-community/molecule-azure
+          - ansible-community/molecule-digitalocean
+          - ansible-community/molecule-ec2
+          - ansible-community/molecule-gce
+          - ansible-community/molecule-hetznercloud
+          - ansible-community/molecule-libvirt
+          - ansible-community/molecule-linode
+          - ansible-community/molecule-openstack
+          - ansible-community/molecule-vagrant
```

Comment:
lets first deal with pycontribs/pytest-molecule moving, over adding more molecule project first. This is because we need to keep zuul happy first.  Moving repos behind the back can be problematic